openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx

```diff
@@ -128,23 +156,44 @@ const QueryCard: FC<QueryCardProp> = ({
           title={
             <Space className="font-normal p-y-xs" size={8}>
               <Text>{queryDate}</Text>
-              <Text>{SINGLE_DOT}</Text>
-              <Text>{`${t('label.by-lowercase')} ${query.updatedBy}`}</Text>
+              <Text className="text-gray-400">{PIPE_SYMBOL}</Text>
```

Comment:
Did we use same Pipe symbol as EntityInfo

Reply:
the symbol is the same, but in entity info, we have added it from localization. and we are avoiding that, right? to add symbol in localized file so I have added const for symbol

Reply:
Let's make a note and remove it in next release